account-kit/smart-contracts/src/ma-v2/actions/install-validation/installValidation.ts

```diff
@@ -0,0 +1,160 @@
+import {
+  AccountNotFoundError,
+  IncompatibleClientError,
+  isSmartAccountClient,
+  EntityIdOverrideError,
+  type GetEntryPointFromAccount,
+  type SendUserOperationResult,
+  type UserOperationOverridesParameter,
+  type SmartAccountSigner,
+} from "@aa-sdk/core";
+import { type Address, type Hex, encodeFunctionData, concatHex } from "viem";
+
+import { semiModularAccountBytecodeAbi } from "../../abis/semiModularAccountBytecodeAbi.js";
+import type { HookConfig, ValidationConfig } from "../common/types.js";
+import {
+  serializeValidationConfig,
+  serializeHookConfig,
+  serializeModuleEntity,
+} from "../common/utils.js";
+
+import { type SMAV2AccountClient } from "../../client/client.js";
+import { type SMAV2Account } from "../../account/semiModularAccountV2.js";
+import { DEFAULT_OWNER_ENTITY_ID } from "../../utils.js";
```

Comment:
⚠️ **[eslint]** <[@typescript-eslint/no-unused-vars](https://typescript-eslint.io/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'DEFAULT_OWNER_ENTITY_ID' is defined but never used.
<!-- __reviewdog__:Cg9kOTcwNGIwZTEzODFkOTESBnJkanNvbg== -->